torch/cuda/random.py

```diff
@@ -17,8 +17,10 @@
     "seed",
     "seed_all",
     "initial_seed",
+    "rng_supported_mesh",
 ]
 
+rng_supported_mesh = True
```

Comment:
should we maintain a supported device type list, instead of add a global var in each device impl?
Or, make the list registerable to allow out-of-tree device to register its capability?

Reply:
I guess it could be done either way. Out of tree device can define this attribute from the random package to declare the support for this feature.